components/core/titles/LandingH2.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div class="flex justify-center">
+        <h2 :class="classObject">{{ subTitle }}</h2>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'LandingH2',
+    props: {
+        subTitle: {
```

Comment:
Did you mean `subtitle` instead of `subTitle`? Normally, when there already exists a `title` prop/variable, the `subtitle` is therefore meaningful. Having `title` and `subtitle` implies subordination and that the former is more important than the latter.

Hence, in this case, because this component only contains one prop to define the texts of h2, I would suggest that this prop is simply named `title`.

Reply:
I fixed it, thanks for the advice.